apps/sms/style/sms.css

```diff
@@ -1062,3 +1062,19 @@ html[dir="rtl"] .contact-header aside.pack-end {
 .editable-select-mode [contenteditable] {
   -moz-user-select: text;
 }
+
+#threads-no-messages, .threadlist-is-empty #threads-container {
+  display: none;
+}
+
+#threads-settings-button, .threadlist-is-empty #threads-options-button {
+  display: none !important;
```

Comment:
nit: add a comment explaining we use "!important" to override scoped style added by the gaia-header component